omod/src/main/java/org/openmrs/module/fhir2/web/util/NarrativeUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.web.util;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+
+public class NarrativeUtils {
+	
+	private static final Logger LOGGER = getLogger(NarrativeUtils.class);
```

Comment:
how do you see if we used logger instead of LOGGER

Reply:
Yeah, it's not a constant so I guess using `logger` is better. I'll use `logger` instead.

Reply:
@HerbertYiga should I use `log` instead? I just saw that `log` has been used at other places in the repository.

Reply:
true,use log

Reply:
@HerbertYiga I've made the changes. PTAL.

Reply:
Actually, if I can suggest, just annotate the class with the `@Slf4j` annotation, which will magically generate the `log` variable for you. See [here](https://projectlombok.org/features/log).

Reply:
Oh that's nice! Will do that.

Reply:
Have made the changes.